sensor/CategoryUpdateSensor.js

```diff
@@ -206,6 +213,14 @@ class CategoryUpdateSensor extends Sensor {
 
       sem.on('Policy:CategoryActivated', async (event) => {
         const category = event.category;
+        const domains = await categoryUpdater.getDefaultDomains(category);
```

Comment:
```
if (!categoryUpdater._isCustomizedCategory(category)) {
  await this.updateCategory(category)
}
```
then you don't need to modify updateCategory function. It's better to rename the _isCustomizedCategory function to isCustomizedCategory since it is used outside the CategoryUpdater.js

```

Reply:
Assume there are only customized category and categories from cloud